### ISSUE TITLE
collecting user contributed themes

### ISSUE THREAD

**Issue by jubalh** [lxde/lxqt#721]
today i saw [this](http://intialonso.github.io/) which makes me think:
what is our approach about collecting themes? do we plan to host only official ones or user contributed themes too?
we could:
1. just let themes be spread over the internet or sites like box-look.org/kde-look.org
2. collect them in a git repo "userthemes"
3. list them on the website

**Comment by Vladimir-csp** [comment]
I would suggest sorting out directories for themes first, use standard paths instead of what LXQT uses now.
And also #572

**Comment by paulolieuthier** [comment]
@Vladimir-csp: agreed.

**Comment by jubalh** [comment]
I agree that these are important issues to sort out but dont think its directly relevant to this one

**Comment by paulolieuthier** [comment]
I don't think there should be a repo for that. @intialonso's site is a great start.

**Comment by intialonso** [comment]
Maybe meanwhile can we put a section in the wiki?

This will be temporal, I agree with Vladimir-csp, we need to solve those
issues first. Right now the theming on LXQT is a little frankentsein, we
need something more unified.

2015-07-01 5:56 GMT-05:00 Paulo Lieuthier <notifications@github.com>:
